kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/NamedRange.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.config;
+
+import java.util.Objects;
+import java.util.stream.IntStream;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.kroxylicious.proxy.tag.VisibleForTesting;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Represents the set of integers between two integer endpoints. So the Range
+ * with startInclusive 1 and endExclusive 3 contains the integers 1 and 2.
+ * A Range must be non-empty, endExclusive must be greater than startInclusive
+ */
+public record NamedRange(@NonNull String name,
```

Comment:
```suggestion
public record NamedRange(@NonNull @JsonProperty(required = true) String name,
```

Reply:
I was just checking out of curiosity and noticed this was resolved without a change to the PR. Have we missed a push or is there a reason this shouldn't be required?